.github/workflows/integration.yml

```diff
@@ -269,6 +273,8 @@ jobs:
     strategy:
       matrix:
         bootstrap-version: ['test', 'main']
+    # Mark a failure of `test` bootstrap job as a successful result
+    continue-on-error: ${{ matrix.bootstrap-version == 'test' }}
```

Comment:
Since I don't really know the motivation behind having a separate `'test'` bootstrap version (I think just that it was different than the `'main'` mode for vaults), is there a reason its result should be ignored?  Is this just codifying behaviour we already had, avoiding a flake, or something else?